README.md

```diff
@@ -188,7 +188,12 @@ If you want to specify a structure as starting point for `init_bulk`, you may se
 ```json
 "from_poscar":	true,
 "from_poscar_path":	"....../C_mp-47_conventional.POSCAR",
-```
+```　
```

Comment:
Please notice that the format is not correct.

Reply:
Sorry for that... I just fixed the bug.